pkg/Makefile.am

```diff
@@ -4,19 +4,19 @@ pkg_TESTS = \
 
 TESTS += $(pkg_TESTS)
 
-%.metainfo.xml: %.metainfo.xml.in
+metainfo_in = \
+	$(srcdir)/pkg/sosreport/org.cockpit-project.cockpit-sosreport.metainfo.xml.in \
+	$(srcdir)/pkg/kdump/org.cockpit-project.cockpit-kdump.metainfo.xml.in \
+	$(srcdir)/pkg/selinux/org.cockpit-project.cockpit-selinux.metainfo.xml.in \
+	$(srcdir)/pkg/storaged/org.cockpit-project.cockpit-storaged.metainfo.xml.in \
+	$(srcdir)/pkg/networkmanager/org.cockpit-project.cockpit-networkmanager.metainfo.xml.in \
+	$(NULL)
+
+tmp/build/%.metainfo.xml: $(srcdir)/pkg/%.metainfo.xml.in
```

Comment:
I don't like using `tmp/` for this.  That's usually rather stuff that happens from our bots-type infra, not on `make`.

Is there a problem with having them in `pkg/`?  In out-of-tree builds that should end up in `pkg/` in the build directory, no?

Reply:
Sorry.  I misunderstood that this is about dependencies, not what gets in the tarball (although it's probably also valid for this reason).

This is tricky.  Output belongs beside .in files, truly.  Why not move this all somewhere else like a new subdir in src/?

Reply:
Done. PTAL?